pkg/cloudprovider/providers/aws/aws_routes.go

```diff
@@ -49,6 +49,12 @@ func (s *AWSCloud) findRouteTable(clusterName string) (*ec2.RouteTable, error) {
 
 // ListRoutes implements Routes.ListRoutes
 // List all routes that match the filter
+func getInstanceAndCIDR(r *ec2.Route)(string, string) {
```

Comment:
My personal opinion is that splitting this function out just adds complexity.  Is there a reason why you want to / have to do it?  If you want to keep it, the comment for ListRoutes needs to be moved to be next to ListRoutes.